File: C++/1149.cpp
#include<iostream>
using namespace std;

int main(){
    int entrada, a = 0, n, soma = 0;

    while(true){
        cin >> entrada;

        if(a == 0){
            if(entrada > 0){
                a = entrada;
            }
        }else{
            if(entrada > 0){
                n = entrada;
                break;
            }
        }
    }

    for(int i = 0; i <= n - 1; i++){
        soma += a + i;
    }

    cout << soma << endl;
    
    return 0;
}

File: JavaScript/1149.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

let a = 0, n, soma = 0;

let entrada = lines[0].split(' ');

var numberArray = [];

for(let i in entrada){
    numberArray.push(parseInt(entrada[i]));
}

[a, n] = numberArray.filter(item => item > 0);

for(let j = 0; j <= n - 1; j++){
    soma += parseInt(a + j);
}

console.log(soma);